core/src/main/java/org/infinispan/util/CoreBlockHoundIntegration.java

```diff
@@ -106,9 +106,8 @@ private static void methodsToBeRemoved(BlockHound.Builder builder) {
       // https://issues.redhat.com/browse/ISPN-11272
       builder.allowBlockingCallsInside(RecoveryManagerImpl.class.getName(), "registerInDoubtTransaction");
 
-      // Scattered prefetch iteration is currently blocking - needs to be rewritten to be non blocking
-      // https://issues.redhat.com/browse/ISPN-10864
-      builder.allowBlockingCallsInside(PrefetchInterceptor.class.getName() + "$BackingIterator", "hasNext");
+      // Scattered state consumer is currently blocking - needs to be rewritten to be non blocking
```

Comment:
I would say to keep the JIRA still there. I made sure for all entries in the removed method that they have a JIRA logged so we can track and keep ideas of how to resolve, since the goal is to not have this long term.

Reply:
@wburns good point. I removed the ISPN-10864 thinking it was strictly related to the prefetch iteration but I see now it's about state transfer and the prefetch iteration was only part of it.

Reply:
So there are no actual code changes?

Reply:
I had reverted the only code change by accident. It's fixed now.